wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -166,9 +166,16 @@ public static function collectionCampTabset($tabsetName, &$tabs, $context) {
         continue;
       }
 
-      $hasPermission = \CRM_Core_Permission::check($config['permissions']);
+      $hasPermission = false;
+      foreach ($config['permissions'] as $permission) {
+          if (\CRM_Core_Permission::check($permission)) {
+              $hasPermission = true;
+              break;
+          }
+      }
+      
```

Comment:
Is it possible to get rid of the `foreach` loop because [`check` accepts permissions as array](https://github.com/coloredcow/goonj/blob/1a259d6e3d70ae7a5010268d9519314c6950c2b4/wp-content/plugins/civicrm/civicrm/CRM/Core/Permission.php#L102-L103) also
```suggestion
        if (!\CRM_Core_Permission::check($config['permissions'])) {
            // does not permission; just continue
            continue;
        }
```

Reply:
Or if that does not work maybe use [`checkAnyPerm`](https://github.com/coloredcow/goonj/blob/1a259d6e3d70ae7a5010268d9519314c6950c2b4/wp-content/plugins/civicrm/civicrm/CRM/Core/Permission.php#L159)